common/src/stack/switch/command/list/switch/partition/member/imp_m7800.py

```diff
@@ -0,0 +1,72 @@
+# @copyright@
+# Copyright (c) 2006 - 2019 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+import stack.commands
+from stack import api
+from stack.switch.m7800 import SwitchMellanoxM7800
+
+class Implementation(stack.commands.Implementation):
+	def get_host_interface_by_mac(self, mac):
+		# Return the host name, interface name, and address
+		# if a given mac belongs to a host in the database
+		host = ''
+		interface_name = ''
+		address = ''
+		for interface in api.Call('list host interface'):
+			# If a mac addresses matches what's in the database
+			# Get it's host name and interface name
+			if interface['mac'] and mac.lower() in interface['mac'].lower():
```

Comment:
Is the first part of this supposed to be `if interface.get('mac') ...`? Or is it that interface['mac'] will always be set but could be set to the empty string or None?

Is this using `in` because of the whole extra characters of the MAC address with IB switches? If not, shouldn't this use `==`?